source/facets.toml

```diff
@@ -0,0 +1,13 @@
+[[facets]]
+category = "target_product"
+value = "drivers"
+
+[[facets]]
+category = "target_product"
+value = "drivers"
+[[facets]]
+category = "programming_language"
+value = "java"
+[[facets]]
+category = "programming_language"
+value = "java"
```

Comment:
Do each of these facets (product and language) need to be included twice in this file?

Reply:
@mongoKart - good catch!  No, they should not be.  The source file is correct, so I'll need to reach out to Seung & Cassidy.  I think the script is picking up the docs-java-other values, and with the info you shared about that one we might just need to delete the async driver repo from the source file.

Reply:
these doubled values are erroneous! good catch @mongoKart !

https://github.com/10gen/snooty-scripts/pull/8

created an update to match repo path exactly as is in CSV (no partial matching for `docs-java-other` etc)

Reply:
Thanks so much, @seungpark!  I'll delete these extra 2 lines so Mike can merge, knowing that the script is now updated so the next time we run it we won't have this problem.